app/(dashboard)/instances/components/InstanceForm.tsx

```
@@ -0,0 +1,611 @@
+          const result = schema.find((schemaParam) => {
+            return schemaParam.key === key;
+          });
+
+          switch (result?.type?.toLowerCase()) {
+            case "number":
+              if (data.requestParams[key] === "") break;
+              data.requestParams[key] = Number(data.requestParams[key]);
+              break;
+            case "float64":
+              if (data.requestParams[key] === "") break;
+              const output = Number(data.requestParams[key]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)